cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -294,10 +294,7 @@ func KnownControllers() []string {
 	return ret.List()
 }
 
-var ControllersDisabledByDefault = sets.NewString(
-	"bootstrapsigner",
-	"tokencleaner",
-)
+var ControllersDisabledByDefault = sets.NewString()
```

Comment:
I wouldn't expect these to be running unless the bootstrap authorizer was being used